### PR TITLE
Script taking arguments from command line

### PR DESCRIPTION
**Pull request by dropsrobs**
Changed script#2 to fit advices. 
gsub regex changed to /[/W]+/
conditions changed from .first & .last to start_with? & end_with?

### CODE REVIEW

**Comment by spidzio** [comment]
Awesome...
Now I will give you a final comments.

File: script2.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# even if we duplicate, arg_dup has sill frozen flag
+arg_dup =  ARGV.sort.reverse
+
+arg_dup.each_with_index { |a, i| arg_dup[i] = a.dup }
+
+arg_dup.each do |a|
```
Comment (spidzio) [inline script2.rb]: Good thinking in general, but all above lines can be simplified this way:
```ruby
ARGV.map(&:dup).sort.reverse_each do |arg|
```
And please rename `a` variable to `arg`

File: script2.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  # need to practice with regexes a lot, regex found on some site
+
+  a = a.gsub(/[\W]+/, '_')
```
Comment (spidzio) [inline script2.rb]: you can simply do:
```ruby
a.gsub!(/[\W]+/, '_')
```

File: script2.rb (hunk 1 above)
Comment (dropsrobs) [inline script2.rb]: Right, you shown us that. Thanks, I'll fix that.

File: script2.rb (hunk 2 above)
Comment (spidzio) [inline script2.rb]: Regex can be simplified: `/\W+/`

**Comment by spidzio** [comment]
Rubocop says - no offences
script works 
:+1: 
Every time you get thumb up, you can merge a pull request. :)
So now you can merge it here in github.

To create next feature branch, do as fallows, after merge:
```bash
git checkout master
git fetch
git checkout -b feature/oop-solution
```

**Pull request merged**